build-support/bin/xz/linux/x86_64/5.2.4-1/xz.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
If you're explicitly overwriting `$PATH` in pants, this won't be able to find a `python`?

(But also, I'd be tempted to just drop this into bash :))

Reply:
I did it in bash, then was struck by the fact that there's no useful `readlink` command on OSX unless you implement it yourself. If we can assume there's going to be a `/usr/bin/perl` I would much prefer that, I'll try that now.

Reply:
Between perl and python, would prefer python.

Reply:
Oops, just saw that -- can revert f9f7565 asap. If there's a clear way to do `readlink -f` on OSX I can make a bash script, otherwise would we want to set a hashbang to a specific path in the python script e.g. `#!/usr/bin/python`?